go.mod

```diff
@@ -182,5 +182,5 @@ require (
 replace (
 	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.4.0-sdk-v0.46.0
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	github.com/tendermint/tendermint v0.34.20 => github.com/celestiaorg/celestia-core v1.10.0-tm-v0.34.20
+	github.com/tendermint/tendermint v0.34.20 => github.com/celestiaorg/celestia-core v1.10.0-tm-v0.34.20.0.20221206233119-6117431617c1
```

Comment:
[optional] I assume the snapshot version was used so that PRs could be published simultaneously but reminder that we likely want to merge https://github.com/celestiaorg/celestia-core/pull/893 , cut a release, and use that version here